addons/mail/static/src/discuss/message_pin/common/thread_patch.js

```diff
@@ -13,6 +13,8 @@ patch(Thread.prototype, {
         const { oeType } = ev.target.dataset;
         if (oeType === "pin-menu") {
             this.env.pinMenu?.open();
+        } else {
+            await super.onClickNotification(ev);
```

Comment:
Is that an unrelated bugfix?

Reply:
Yes indeed, the "Jump" was broken because of the missing call to super()

Reply:
I'm still not sure to understand, what is the situation to reproduce this bug?
Can we have a test to cover it?